src/utils/hints/stateChannelsHints.js

```diff
@@ -0,0 +1,9 @@
+export const stateChannelsHints = {
+  stateChannelId: 'Unique identifier of the State Channel.',
+  status: 'When status is \'Open\' transactions can be executed inside the State Channel, outside the blockchain network',
+  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx.',
+  onChainUpdates: 'Amount of off-chain transactions within the State Channel.',
```

Comment:
that looks weird. I would assume that on-chain updates are... well.. on chain :D Is it correct?

Reply:
fixed